fenix/app/metrics.yaml

```diff
@@ -6090,7 +6090,75 @@ logins:
     metadata:
       tags:
         - Logins
-
+  saved:
+    type: counter
+    description: |
+      Counter of number of passwords that have been saved by user (including deleted).
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1836164
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-android/pull/2555
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+      - cgordon@mozilla.com
+    expires: never
+    metadata:
+      tags:
+        - Logins
+  saved_all:
+    type: quantity
+    description: |
+      Counter of number of passwords currently saved by user.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1836164
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-android/pull/2555
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+      - cgordon@mozilla.com
+    expires: never
+    unit: integer
+    metadata:
+      tags:
+        - Logins
+  deleted:
+    type: counter
+    description: |
+      Counter of number of passwords that have been deleted by user.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1836164
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-android/pull/2555
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+      - cgordon@mozilla.com
+    expires: never
+    metadata:
+      tags:
+        - Logins
+  modified:
+    type: counter
+    description: |
+      Counter of number of passwords that have been modified by user.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1836164
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-android/pull/2555
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+      - cgordon@mozilla.com
+    expires: never
+    metadata:
+      tags:
+        - Logins
```

Comment:
🔍 nit: space after this to separate from `voice_search`?

Reply:
sure, I'll add a space